### DISCUSSION TITLE
[Vote ended on 2022-10-04] Remove dellemc.os6 from Ansible 8

### DISCUSSION THREAD

**Discussion by mariolenz** [ansible-community/community-topics#139] [Votes done]
`dellemc.os6` looks unmaintained (#132).

Please vote with a +1 to remove it from Ansible 8 or -1 to keep it in this discussion.

**Comment by mariolenz** [comment]
+1

**Comment by felixfontein** [comment]
Considering that there was no reaction to https://github.com/ansible-collections/dellemc.os6/issues/54 in a month: +1

**Comment by markuman** [comment]
+1

**Comment by russoz** [comment]
+1

**Comment by cybette** [comment]
+1

**Comment by briantist** [comment]
+1

**Comment by ssbarnea** [comment]
+1

**Comment by Andersson007** [comment]
+1

**Comment by gundalow** [comment]
+1

**Comment by acozine** [comment]
+1
Do we know why the collections got dropped? Are they for EOL versions?
